data/json/items/tool_armor.json

```diff
@@ -1352,7 +1352,12 @@
         "flags" : ["VARSIZE", "WATER_FRIENDLY", "STURDY", "WAIST", "OVERSIZE"],
         "coverage" : 15,
         "material_thickness" : 2,
-        "use_action": [ "SHEATH_KNIFE", "HAMMER", "HACKSAW", "LUMBER", "REMOVE_ALL_MODS" ],
```

Comment:
See commit notes - it's a temporary break as I don't think you can mix `iuse_actor` with `iuse` functions. Unless I'm incorrect (maybe?) the options would be to extend parsing of `use_action` or implement a custom menu for the survivor utility belt